robot-java/src/main/java/dc/Robot.java

```diff
@@ -157,7 +154,7 @@ public void autonomousPeriodic() {
 		priorAutospeed = autospeed;
 
 		// command motors to do things
-drive.tankDrive(autospeed, autospeed, false);
+		drive.tankDrive(autospeed, autospeed, false);
```

Comment:
Why did this file change? It's not relevant to your PR.

Reply:
My bad, I probably auto-formatted it by accident; I'll revert this change.